support/spray/src/test/scala/sjsonnew/support/spray/JavaExtraFormatsSpec.scala

```diff
@@ -79,14 +79,14 @@ class JavaExtraFormatsSpec extends Specification with BasicJsonProtocol {
     val f = new File("/tmp")
     val f2 = new File(new File("src"), "main")
     "convert a File to JsString" in {
-      Converter.toJsonUnsafe(f) mustEqual JsString("file:///tmp/")
+      Converter.toJsonUnsafe(f) mustEqual JsString("file:///tmp")
```

Comment:
Not sure how much we should be concerned about the change in the string representation here. Hopefully it's ok but I wanted to note this change so we remember it.

Reply:
Well from my tests in Windows that I shared above you can see we can still correctly read the String without "/" in the end to a `File` and that correctly represents a directory. Since in sbt code we will only handle `File`s already deserialized I think it should be okay.